dev-docs/modules/userId.md

```diff
@@ -1058,7 +1058,7 @@ In either case, bid adapters will receive the eid values after consent is valida
 
 - dmp - this uid comes from the in-page dmp named in eids.source
 - ppuid - this uid comes from the publisher named in eids.source
-- other - TBD
+- other - for setting other id origin signals please use the [adCOM!](https://github.com/InteractiveAdvertisingBureau/AdCOM/blob/master/AdCOM%20v1.0%20FINAL.md#list_agenttypes) `atype` spec 
```

Comment:
Since you refer to the atype spec, wouldn't it be clearer to link to the reference to atype just to get the user oriented, where they can then see the link to the list? https://github.com/InteractiveAdvertisingBureau/AdCOM/blob/master/AdCOM%20v1.0%20FINAL.md#object--extended-identifier-uids-

Reply:
I don't have a strong opinion either way.